assets/js/main.js

```diff
@@ -18,11 +18,16 @@ let earthCurrentTitleTimezone;
 let earthCurrentIndicator;
 let earthCurrentIndicatorColor;
 
-// Packery
-let grid;
-
 const fissureGlyphs = ['https://i.imgur.com/D595KoY.png', 'https://i.imgur.com/VpBDaZV.png', 'https://i.imgur.com/YOjBckN.png', 'https://i.imgur.com/nZ3FfpC.png'];
 
+const defaultLayout = [
+  ['cetus'], ['earth'],
```

Comment:
please keep acolytes enabled by default at the top

Reply:
Acolytes are still on top. It doesn't allow to move them so that component is not included in layout.

Reply:
hmmm, but what if i want it, just further down?

Reply:
is the only part preventing it that it's wider than everything else?

Reply:
Yes. **dragula** is moving components from one column to another (`.col-md-6`). And since Acolytes are in bigger column I just left it as is.

Reply:
`.col-md-6` is just the size most of them use... like if i used a `.col-md-4` this should still work